haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -384,9 +386,18 @@ newtype DynamicRec n = DynamicRec { unDynamic :: Map MText ByteString }
 dynRecUnsafe :: DynamicRec n
 dynRecUnsafe = DynamicRec mempty
 
+-- | The same as `DynamicRec` but uses `BigMap` instead.
+newtype DynamicRecBigMap n = DynamicRecBigMap { unDynamicBigMap :: BigMap MText ByteString }
+  deriving stock (Generic, Show, Eq)
+  deriving newtype (IsoValue, HasAnnotation, Default, One, Semigroup)
+
+-- | Construct 'DynamicRecBigMap' assuming it contains no mandatory entries.
+dynRecBigMapUnsafe :: DynamicRecBigMap n
+dynRecBigMapUnsafe = DynamicRecBigMap mempty
+
 -- TODO consider making these all 'BigMap's instead
```

Comment:
Could you please remove this comment?

I just created #192 for `CustomEntrypoints` and we have this issue and #189 for `ContractExtra` and `ProposalMetadata` respectively, so the "consider" part is done, I think :slightly_smiling_face:

Reply:
Removed this